gradlew

```diff
@@ -1,18 +1,7 @@
 #!/bin/sh
 
 #
-# SPDX-License-Identifier: Apache-2.0
-#
-# The OpenSearch Contributors require contributions made to
-# this file be licensed under the Apache-2.0 license or a
-# compatible open source license.
-#
-# Modifications Copyright OpenSearch Contributors. See
-# GitHub history for details.
-#
-
-#
-# Copyright 2015-2021 the original author or authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
We should re-add these licenses. Thoughts?

Reply:
Definitely not, those are 100% from Gradle, we do not own them or modified them

Reply:
May be during the migration, or they were modified, ...